apps/ai-content-generator/src/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { StrictMode } from 'react';
+import { render } from 'react-dom';
+
+import { GlobalStyles } from '@contentful/f36-components';
+import { SDKProvider } from '@contentful/react-apps-toolkit';
+
+import LocalhostWarning from './components/LocalhostWarning';
+import App from './App';
+
+const root = document.getElementById('root');
```

Comment:
I believe this API has been updated to render the root like this now adays, but not necessary just mor e a call out

```typescript
const root = ReactDOM.createRoot(document.getElementById('root'));
root.render(
  <React.StrictMode>
    <App />
  </React.StrictMode>
);
```

Reply:
💩 :shovel: – will fix – PR into this welcome

Reply:
I don't think its critical, this is just propagated into the templates these days 👍